### ISSUE TITLE
can't download jrosetta

### ISSUE THREAD

**Issue by Alessandro-Barbieri** [gentoo-science/sci#412]
I get 404 not found error when fetching jrosetta as a dependency for scilab.

```
>>> Emerging (1 of 66) dev-java/jrosetta-1.0.4::science
 * Fetching files in the background. To view fetch progress, run
 * `tail -f /var/log/emerge-fetch.log` in another terminal.
>>> Downloading 'http://mirror.switch.ch/ftp/mirror/gentoo/distfiles/jrosetta-1.0.4-sources.jar'
--2015-05-05 00:39:23--  http://mirror.switch.ch/ftp/mirror/gentoo/distfiles/jrosetta-1.0.4-sources.jar
Resolving mirror.switch.ch... 130.59.10.36, 2001:620:0:8::20
Connecting to mirror.switch.ch|130.59.10.36|:80... connected.
HTTP request sent, awaiting response... 404 Not Found
2015-05-05 00:39:23 ERROR 404: Not Found.

>>> Downloading 'http://maven.artenum.com/content/groups/public/com/artenum/jrosetta/1.0.4/jrosetta-1.0.4-sources.jar'
--2015-05-05 00:39:23--  http://maven.artenum.com/content/groups/public/com/artenum/jrosetta/1.0.4/jrosetta-1.0.4-sources.jar
Resolving maven.artenum.com... 188.165.220.48
Connecting to maven.artenum.com|188.165.220.48|:80... connected.
HTTP request sent, awaiting response... 404 Not Found
2015-05-05 00:39:23 ERROR 404: Not Found.

!!! Couldn't download 'jrosetta-1.0.4-sources.jar'. Aborting.
 * Fetch failed for 'dev-java/jrosetta-1.0.4', Log file:
 *  '/var/tmp/portage/dev-java/jrosetta-1.0.4/temp/build.log'

>>> Failed to emerge dev-java/jrosetta-1.0.4, Log file:

>>>  '/var/tmp/portage/dev-java/jrosetta-1.0.4/temp/build.log'

 * Messages for package dev-java/jrosetta-1.0.4:

 * Fetch failed for 'dev-java/jrosetta-1.0.4', Log file:
 *  '/var/tmp/portage/dev-java/jrosetta-1.0.4/temp/build.log'
```

**Comment by marbre** [comment]
This is an upstream problem: The link mentioned on http://dev.artenum.com/projects/JRosetta/download/JRosetta-1-0-4 to the jrosetta-1.0.4-sources.jar gives:

404 - Not Found
The server has not found anything matching the request URI

We can't solve this.

**Comment by Alessandro-Barbieri** [comment]
I tried 2 other mirrors and noone is mirroring jrosetta.

**Comment by marbre** [comment]
I currently don't have time to contact upstream, maybe you would like to do that. If we find no way to get the source, the only option is to drop the packages, which would be bad as other packages AFAIK depend on jrosetta.

**Comment by thrasibule** [comment]
I just asked on the scilab mailing list if they know how to get in touch with upstream. It looks like they're the only user, so they might want to take jrosetta under their umbrella.

**Comment by Alessandro-Barbieri** [comment]
Sent a mail to upstream, awaiting response.

**Comment by JohannesBuchner** [comment]
You can get it here. Switch the uri or perhaps gentoo(-science) can host it too?
http://pkgs.fedoraproject.org/repo/pkgs/jrosetta/jrosetta-1.0.4-sources.jar/5c3589d4207f71bad6eeefd4857bce50/jrosetta-1.0.4-sources.jar

**Comment by thrasibule** [comment]
This is what I got from the scilab developers: http://mailinglists.scilab.org/jrosetta-sources-not-available-td4032218.html Looks like upstream is not dead, they should bring the download link back up. In the meantime the fedora sources is a good option.

**Comment by Alessandro-Barbieri** [comment]
Their website went down some days ago. They brought it up, but no response since I contacted.

**Comment by JohannesBuchner** [comment]
Please merge to solve this issue temporarily until the upstream server comes back.

**Comment by Alessandro-Barbieri** [comment]
The link
http://pkgs.fedoraproject.org/repo/pkgs/jrosetta/jrosetta-1.0.4-sources.jar
is invalid: points to an html file

this is the right link
http://pkgs.fedoraproject.org/repo/pkgs/jrosetta/jrosetta-1.0.4-sources.jar/5c3589d4207f71bad6eeefd4857bce50/jrosetta-1.0.4-sources.jar

**Comment by marbre** [comment]
Thanks for the hint. Fixed with commit 09cbce583c1f59dbf514dd37c1dc9a18b054c924.
